### DISCUSSION TITLE
dump-env called before deploy:vendors with Symfony recipe

### DISCUSSION THREAD

**Discussion by YetiCGN** [deployphp/deployer#3938] [Bugs] [answered]
### Deployer Version

v7.5.x

### Target OS

Debian 12 Bookworm, but doesn't matter

### Which PHP version are you using?

PHP 8.3

### Content of deploy.php or deploy.yaml
```php
<?php declare(strict_types=1);

namespace Deployer;

require 'recipe/symfony.php';

set('application', 'lorem-ipsum);
set('allow_anonymous_stats', false);
set('release_name', function () {
    return date('YmdHis');
});
set('keep_releases', 5);
set('shared_dirs', [
    'var/log',
    'var/media',
]);

set('shared_files', [
    '.env.local',
    '.env.test.local'
]);

// Hosts
host('dev')
    ->setHostname('deploy-host.dev')
    ->setRemoteUser('remote-user')
    ->set('stage', 'develop')
    ->setDeployPath('/var/www')
    ->set('branch', 'develop')
;
```

### Steps to reproduce

Upgrading from 7.4 to 7.5, our deployment pipeline fails during `deploy:env`, which was newly introduced (or reintroduced? see #2517).

Because the `vendor` folder is ignored during the previous task `deploy:update_code`, there is no Flex plugin from Symfony for this command to successfully execute. This will only be installed later with `deploy:vendors`. 

I guess the task should be moved further back in the pipeline or the Symfony recipe needs to insert `deploy:vendors` between `update_code` and `env`. 

For the time being we are overriding `deploy:env` to not do anything.

**Comment by antonmedv** [comment]
Yes, this was unfarcical override by deploy:env.

**Comment by antonmedv** [comment, marked as answer]
Fixed.
